composer/models/resnet/model.py

```diff
@@ -92,7 +93,7 @@ def composer_resnet(model_name: str,
         weights = 'IMAGENET1K_V2'
         warnings.warn(
             DeprecationWarning(
-                'The ``pretrained`` argument for composer_resnet is deprecated and will be removed in the future. Please use ``weights`` instead.'
+                'The ``pretrained`` argument for composer_resnet is deprecated and will be removed in the future when torch 1.11 is no longer supported. Please use ``weights`` instead.'
```

Comment:
FYI: Due to security vulnerabilities, we will likely deprecate Torch 1.11 support in the next release, minor or micro whatever comes first.